.changes/unreleased/Features-20240318-033621.yaml

```
@@ -0,0 +1,6 @@
+kind: Features
+body: ': Add new workflow for internal patch releases'
```

If adapters wants these changelogs
```suggestion
body: 'Add new workflow for internal patch releases'
```